examples/energy_mng/ui/theme.slint

```diff
@@ -28,22 +52,59 @@ export struct Spaces {
 export struct Durations {
     fast: duration,
     medium: duration,
+    slow: duration,
+}
+
+struct Font {
+    size: length,
+    weight: float,
+}
+
+export struct Typo {
+    label: Font,
+    label-light: Font,
+    header: Font,
+    value: Font,
 }
 
 export global Theme {
     in property <Palette> palette: {
-        // solid
+        // primary
         slint-blue: #0025FF,
-        purple-black: #000000,
+        pure-black: #000000,
         dark-deep-blue: #040708,
         shark-gray: #2C2F36,
         limon-green: #DEFB3A, 
         white: #FFFFFF,
+ 
+        // slint blue
+        slint-blue-50: #EEE6FF,
```

Comment:
Do we really need all this in the palette?
Maybe this can  be done in the usage size with  `paltte.slint-blue.darker(10%)`  or something like that

Reply:
The palette is complete defined on the figma design. I think it make sense to use the concrete values from the design, this makes it easier to avoid mistakes and is easier to check.